cmd/server/main.go

```diff
@@ -7,186 +7,305 @@ import (
 	_ "github.com/mattn/go-sqlite3"
 	"log"
 	"net/http"
-	"os"
+	"path/filepath"
 	"strconv"
 )
 
 type product struct {
-	ID          int    `json:"id"`
-	Name        string `json:"name"`
-	Description string `json:"description"`
-	Price       int    `json:"price"`
+	ID          int     `json:"id"`
+	Name        string  `json:"name"`
+	Description string  `json:"description"`
+	Price       float64 `json:"price"`
 }
 
 var products []product
 
-func createDB() {
-	os.Remove("./products.db")
-	db, err := sql.Open("sqlite3", "./products.db")
-	if err != nil {
-		log.Fatal(err)
-	}
-	defer db.Close()
+var db *sql.DB
+var err error
 
-	sqlStmt := `
-		CREATE TABLE products_table (ID INTEGER NOT NULL PRIMARY KEY, name TEXT, description TEXT, price REAL);`
-	_, err = db.Exec(sqlStmt)
+func connectDB() *sql.DB {
+	dbPath, err := filepath.Abs("../../cmd/seeder/products.db")
```

Comment:
Would have been best to parameterize this and use a flag to obtain the location of the DB file on startup.